macos/FluentUITestApp/TestLinkViewController.swift

```diff
@@ -4,12 +4,12 @@
 //
 
 import AppKit
+import FluentUI
 
-class TestHoverButtonViewController: NSViewController {
+class TestLinkViewController: NSViewController {
 	override func loadView() {		
 		let containerView = NSStackView(frame: .zero)
```

Comment:
as a very minor point to help make the UI declarative (I know I started this bad pattern) - more standard would be to do something like
`let link = Link(title: "Bing", url: NSURL(string: "https://www.bing.com")!)
let containerView = NSStackView(views: [link])

Also are there any other types of links we want to add here for illustrative purposes? Maybe one where the client customizes the target (and shows an alert) if we agree that's a thing they should be able to do?
`

Reply:
I added a link with an optional hover effect, and I'll add one more with an overriden target/action. Maybe also some links inside text to show how they don't wrap when they reach the edge of a paragraph (this is intended)